app/components/course-page/course-stage-step/language-guide-card.hbs

```diff
@@ -57,42 +57,22 @@
         <span class="text-gray-500">Proficiency:</span>
         <span class="text-gray-700 font-semibold">Beginner</span>
       </div> }}
-      <div>
-        <LanguageDropdown
-          @languages={{this.sortedLanguagesForDropdown}}
-          @requestedLanguage={{this.currentLanguage}}
-          @selectedLanguage={{this.currentLanguage}}
-          @onRequestedLanguageChange={{this.handleRequestedLanguageChange}}
-          @onDidInsertDropdown={{(noop)}}
-          class="flex-shrink-0"
-        />
-      </div>
+
+      <FeedbackButton
```

Comment:
Let's not touch this component and instead add `CoursePage::CourseStageStep::SimpleLanguageGuideCard`. It can accept `languageGuide` directly, instead of taking `repository` and `courseStage`. 

Once our experiment is proven to work, we'll revert back to using this.